cmd/webhook/main.go

```diff
@@ -65,9 +68,28 @@ func main() {
 	// Load webhook config
 	c := wh.LoadConfig(*sidecarImage, *imagePullPolicy, *cpuRequest, *cpuLimit, *memoryRequest, *memoryLimit, *ephemeralStorageRequest, *ephemeralStorageLimit)
 
+	// Load config for manager, informers, listers
+	kubeConfig := config.GetConfigOrDie()
+
+	// Setup client
+	client, err := kubernetes.NewForConfig(kubeConfig)
+	if err != nil {
+		klog.Fatalf("Unable to get clientset: %v", err)
+	}
+
+	// Setup stop channel
+	stopCh := signals.SetupSignalHandler()
+
+	// Setup Informer
+	informerFactory := informers.NewSharedInformerFactory(client, time.Duration(1))
+	nodeLister := informerFactory.Core().V1().Nodes().Lister()
+
+	informerFactory.Start(stopCh.Done())
+	informerFactory.WaitForCacheSync(stopCh.Done())
```

Comment:
I wonder if we need the cache sync, some other code snippets I've seen do not use this. However, My unit tests failed when this was missing, so it might not hurt to include.

Reply:
Yeah I think it makes sense to have this wait to populate the informer before the webhook starts to accept requests.